lib/dfe_signin/utils.rb

```diff
@@ -0,0 +1,14 @@
+module DfeSignin
+  class Utils
```

Comment:
How about if this was `module Utils`, rather than a class? That way there wouldn't be any need for the class method stuff.

Reply:
Good plan - done! 👍